http-server/src/main/java/io/airlift/http/server/HttpServerProvider.java

```diff
@@ -70,9 +67,9 @@ public HttpServerProvider(HttpServerInfo httpServerInfo,
             NodeInfo nodeInfo,
             HttpServerConfig config,
             Optional<HttpsConfig> httpsConfig,
-            @TheServlet Servlet theServlet,
-            @TheServlet Set<Filter> filters,
-            @TheServlet Set<HttpResourceBinding> resources,
+            Servlet theServlet,
+            Set<Filter> filters,
+            Set<HttpResourceBinding> resources,
```

Comment:
It feels a bit strange to inject these `jakarta.servlet` directly without an annotation, but we own the injector and I can't think of any reason why application code would inject these, so I guess it's fine.

Reply:
Actually it's a preparation to reintroduce annotated Servlets, Filters along with the binder for HttpServer :)